bin/startOpfab.sh

```
@@ -0,0 +1,77 @@
+# SPDX-License-Identifier: MPL-2.0
+# This file is part of the OperatorFabric project.
+
+(
+
+    MODE=${1:-dev}
+    
+    # create a directory to store the current configuration 
+    # if not existing
+    
+    mkdir -p ../config/current
+    cd ../config/current
```

Shellcheck (0.10.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)